src/app/components/PortraitVideoModal/index.styles.tsx

```diff
@@ -18,10 +19,15 @@ const styles = {
       flexDirection: 'column',
       alignItems: 'center',
       justifyContent: 'center',
+      zIndex: 2147483647,
```

Comment:
Lol where did this number come from 😆

Reply:
Its the maximum value `zIndex` can be, which is the maximum a 32bit int can be.